examples/examples.json

```diff
@@ -113,6 +113,21 @@
                 }
             ]
         },
+        {
+            "title": "Sprites",
```

Comment:
I'm not sure about creating a separate bucket just for sprites... in my opinion, it should be part of Styling.

Reply:
:+1: